src/index.js

```diff
@@ -123,7 +123,7 @@ function showTasks() {
   free(div);
   const c = this.children;
   let txt = '';
-  for (let i = 0; i < c.length; i = i + 1) {
+  for (let i = 0; i < c.length; i = i + 1) { // eslint-disable-line
```

Comment:
- Use ES6 syntax throughout this module. You still have functions declared with the `function` keyword. Functions like `addTask`, `deleteProject`, etc
- Do the same with functions in `storage.js` module.